docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -283,13 +283,6 @@ Outline styles.
 | style | string, object |  |
 | width | string, object |  |
 
----
-
-### css
-
-Sets custom CSS to apply styling not covered by other theme.json properties.
-
-
```

Comment:
If the schema is strictly defined, the `css` property will unfortunately be removed from the document. It may be possible to revive this description by slightly forcing the `docs:theme-ref` logic to be adjusted.

However, now that `css` properties are only allowed in certain hierarchies, I personally think it is acceptable. If in the future, `css` properties will be available everywhere related to style, I think it would be fine to add them at that time.